svelte.config.js

```diff
@@ -3,15 +3,28 @@ dotenv.config();
 
 import adapterNode from '@sveltejs/adapter-node';
 import adapterCloudflare from '@sveltejs/adapter-cloudflare';
+import adapterStatic from '@sveltejs/adapter-static';
 import { vitePreprocess } from '@sveltejs/vite-plugin-svelte';
 
-const adapterConfig = {
-	// See below for an explanation of these options
-	routes: {
-		include: ['/*'],
-		exclude: ['<all>']
-	}
-};
+let adapter;
+if (process.env.ADAPTER === 'electron-node') {
+	adapter = adapterStatic({
+		pages: 'build',
+		assets: 'build',
+		fallback: 'index.html'
+	});
+} else {
+	const adapterConfig = {
+		// See below for an explanation of these options
+		routes: {
+			include: ['/*'],
+			exclude: ['<all>']
+		}
+	};
+
+	if (process.env.ADAPTER === 'docker-node') adapter = adapterNode(adapterConfig);
```

Comment:
> We have 2 build pipelines: `adapter-node` and `adapter-cloudflare`, we use [`adapter-node` for the Docker release](https://github.com/fmaclen/hollama/blob/main/svelte.config.js#L27-L29) and we would also use that one for Electron.

```typescript
if (['docker-node', 'electron-node'].includes(process.env.ADAPTER)) adapter = adapterNode(adapterConfig);
```

Reply:
About this, I couldn't make Electron work using `adapter-node`, that's why I used `adapter-static`. I can give it another try.

Reply:
Canutin desktop [works with `adapter-node`](https://github.com/fmaclen/canutin-desktop/blob/master/sveltekit/svelte.config.js) so this one should too; but it's possible I'm missing some other required config.

Did you run `ADAPTER="electron-node" npm run build`?
Maybe the build you tried used was made with `adapter-cloudflare` (which is the default adapter)?

Make the change, let me know what errors you are getting and I'll give you a hand.